### ISSUE TITLE
Traceback when attempting to export NIF with keyframes

### ISSUE THREAD

**Issue by Chronoculars** [Greatness7/io_scene_mw#4]
When attempting to export to .NIF a mesh I have animated, I recieve the following error.
```
Python: Traceback (most recent call last):
  File "C:\Users\me\AppData\Roaming\Blender Foundation\Blender\2.91\scripts\addons\io_scene_mw\__init__.py", line 466, in execute
    return nif_export.save(context, **kwargs)
  File "C:\Users\me\AppData\Roaming\Blender Foundation\Blender\2.91\scripts\addons\io_scene_mw\nif_export.py", line 34, in save
    exporter.execute()
  File "C:\Users\me\AppData\Roaming\Blender Foundation\Blender\2.91\scripts\addons\io_scene_mw\nif_export.py", line 81, in execute
    node.animation.create()
  File "C:\Users\me\AppData\Roaming\Blender Foundation\Blender\2.91\scripts\addons\io_scene_mw\nif_export.py", line 1049, in create
    self.create_translations(fcurves_dict)
  File "C:\Users\me\AppData\Roaming\Blender Foundation\Blender\2.91\scripts\addons\io_scene_mw\nif_export.py", line 1111, in create_translations
    keys = self.collect_keyframe_points(fcurves, key_type, num_axes=3)
  File "C:\Users\me\AppData\Roaming\Blender Foundation\Blender\2.91\scripts\addons\io_scene_mw\nif_export.py", line 1443, in collect_keyframe_points
    assert len(fcurves) == num_axes
AssertionError

location: <unknown location>:-1
```
The animations done were exclusively Z position translations. The setting to "Extract Keyframe Data" was checked when attempting to export. The files involved can be found [here.](https://drive.google.com/file/d/1icjPjggwnD1pq8iT1Nqn0lmUhTZqkFLf/view?usp=sharing)

This error occured on Windows 10, with Blender 2.91, io_scene_mw 0.8.53, and Python 3.8.5.

[edit: spelling]

**Comment by Greatness7** [comment]
The NIF format expects the full location vector to be present in keyframe data, rather than just the Z axis. In the future I plan to handle this more gracefully, but for now you can get around your export problem by providing keyframes for the X and Y axes in addition to Z the axis. (even if the X/Y values are just zeros)

**Comment by Greatness7** [comment]
Should be fixed now, as of version `0.8.65`.
